Makefile

```diff
@@ -82,8 +82,8 @@ SKIP_DOCKER_COMMAND ?= false
 GLOBAL_OPERATOR_NAMESPACE ?= elastic-system
 # namespace in which the namespace operator is deployed (see config/namespace-operator)
 NAMESPACE_OPERATOR_NAMESPACE ?= elastic-namespace-operators
-# namespace in which the namespace operator should watch resources
-MANAGED_NAMESPACE ?= default
+# namespaces in which the namespace operator should watch resources
```

Comment:
WDYT to comment it's a comma separated list and then, just use `$(MANAGED_NAMESPACES)` in the Makefile (without `echo .. | tr ..`).

Reply:
:+1:  That would make things easier for sure.  I was hesitant to deviate from the standard way of defining lists but I think it's acceptable here.